docs/plots/index.rst

```diff
@@ -300,12 +300,7 @@ line position (which corresponds to `mdl.pars[2]`):
     >>> from sherpa.plot import IntervalProjection
     >>> iproj = IntervalProjection()
     >>> iproj.calc(f, mdl.pars[2])
-    WARNING: hard minimum hit for parameter base.c0
-    WARNING: hard maximum hit for parameter base.c0
-    WARNING: hard minimum hit for parameter line.fwhm
-    WARNING: hard maximum hit for parameter line.fwhm
-    WARNING: hard minimum hit for parameter line.ampl
-    WARNING: hard maximum hit for parameter line.ampl
+    ...
```

Comment:
This change is here for two reasons: 

1. We actually talk about plotting changes with a pretty dumb data setup. I don't want to discuss *why* we get these warnings and why they can be ignored.
2. Our warnings are not emitted by the warnings module (which captures in doctests and the output chan be compared using pytest-doctestplus) but by a Logger at the warning level. Thus, it's non-trivial to make that work for a doctest and I think the advantage of testing (and finding e.g. the spelling error in "TemporarypPlottingBackend" above) is bigger than displaying these warnings that are meaningless in the context of this example.

Reply:
I think when I added this text I didn't really grok the fact we can use "..." to say "skip text when testing"